### ISSUE TITLE
github: workflow: GitHub Actions comment bot to help users check CI failures

### ISSUE THREAD

**Issue by johnandersen777** [intel/dffml#1262]
Reference the below link for the how to read the CI section

https://github.com/intel/dffml/blob/master/docs/contributing/git.rst#how-to-read-the-ci

this section should be converted from restructuredtext to markdown

the new markdown document should be included under the index.rst table of contents within the docs/contributing directory

in GitHub actions workflow should be created which looks for completions of CI jobs for pull requests

the action should check if there are any failing jobs if there are not any failing jobs the action should do nothing

if there are any failing jobs the action should create a comment containing the following information

- [ ] the contents of the new markdown file which describes how to read the CI

- [ ] the action should check the failing CI jobs against the list of issues labeled with kind/ci/failing. If there are any entries in both less the workflow should provide a message saying that these CI jobs might possibly be failing for these known issues

- [ ] finally the workflow should list each failing job with a link to the report

**Comment by deepakdinesh1123** [comment]
Can I work on this?

**Comment by programmer290399** [comment]
@deepakdinesh1123  Yes! Please see https://intel.github.io/dffml/master/contributing/dev_community.html#who-s-working-on-what. As a rule we don't ask if we can do something, we state our intention to try to do it, then follow those guidelines.

**Comment by deepakdinesh1123** [comment]
Thank you for letting me contribute to the repository, I am working on the issue, I will make a pull request soon.

**Comment by deepakdinesh1123** [comment]
I could not find a find a Github action that lists all issues with certain label, it can be done using the API, I also tried to look for it in the marketplace but was unable to find one. How do I proceed?

**Comment by pdxjohnny** [comment]
I would check out https://github.com/actions/github-script

**Comment by deepakdinesh1123** [comment]
I checked out the repository and it seems like we need to write query using the github api v4 that uses graphql. Unfortunately I do not have enough knowledge about graphql and javascript so I cannot contribute to this issue.
